drivers/impuls-magnetic-mib.c

```diff
@@ -0,0 +1,281 @@
+/*  impuls_magnetic_mib.c - (ietf-mib.c + fix for upsload for IMPULSE SNMP-card for 1-10kVA UPS)  - data to monitor SNMP UPS (RFC 1628 compliant) with NUT
+ *
+ *  Copyright (C) 2002-2006
+ *  2012-2024   Sergei Kartavtsev <sergega@me.com>
+ *	2002-2012	Arnaud Quette <arnaud.quette@free.fr>
+ *	2002-2006	Niels Baggesen <niels@baggesen.net>
+ *	2002-2006	Arjen de Korte <adkorte-guest@alioth.debian.org>
+ *
+ *  Sponsored by MGE UPS SYSTEMS <http://www.mgeups.com>
```

Comment:
This specific driver was not sponsored by MGE :) Was it by the makers of Impuls(e)?